app/scripts/lib/metrics.js

```diff
@@ -153,6 +156,46 @@ define(function (require, exports, module) {
       this._clearInactivityFlushTimeout();
     },
 
+    _initializeFlowEvents (options) {
+      const notifier = options.notifier;
+
+      if (notifier) {
+        notifier.on('flow.initialize', () => this._initializeFlowModel(options.sentryMetrics));
```

Comment:
I find passing options.sentryMetrics a bit strange since `_initializeFlowModel` already accesses `this._flowModel` and `this._window`. Seems like it'd be easier to set `this._sentryMetrics` in the initializer then reference it within `_initializeFlowModel`